### ISSUE TITLE
activation list does not understand trigger activations

### ISSUE THREAD

**Issue by rabbah** [adobe/aio-cli-plugin-runtime#122]
**Describe the bug**
activation list doesn't understand trigger activations.
```
> ... activation list
Datetime              Activation ID                    Kind      Start     Duration    Status    Entity                  
2/12/2020, 9:21:17 AM e436f6ef7bc64661b6f6ef7bc656614e undefined undefined undefinedms undefined undefined               
2/12/2020, 9:20:23 AM a8b8da65cc07464db8da65cc07064db7 undefined undefined undefinedms undefined undefined               
2/12/2020, 9:18:58 AM d90056669ff04a878056669ff09a8772 undefined undefined undefinedms undefined undefined      
```

**To Reproduce**
1. create triggers
2. invoke triggers
3. run activation list

**Expected behavior**
The columns should emit proper values for triggers.

**Comment by rabbah** [comment]
I have a patch here: https://github.com/nimbella-corp/aio-cli-plugin-runtime/commit/f4bf54ba8b7af28749ccd8dbb79d656be850da17 but have not added any tests.

**Comment by rabbah** [comment]
This is now fixed by #207.
